cl/users/templates/includes/docket_alert_webhook_dummy.txt

```diff
@@ -0,0 +1,46 @@
+{
+   "results":[
```

Comment:
Two thoughts. First, could we use Factory Boy here, and if so, would that simplify any of our other tests? It seems like a neat way to send different results each time that wouldn't be too hard.

If that's too much of a pain (or if it also doesn't help our test fixtures), we could at least use this as an advocacy opportunity. For example, here's the PACER fees settlement docket entry: https://www.courtlistener.com/docket/4214664/140/national-veterans-legal-services-program-v-united-states/

(Bill's latest commit/PR has an example of Factory Boy that looks pretty simple.)

Reply:
I ended up here just using https://www.courtlistener.com/docket/4214664/140/national-veterans-legal-services-program-v-united-states/ as an example case. When I was working on generating the JSON using Factory boy I realized that it will have more value for users to have a real docket entry for webhook testing so they can also test processing real content within their system and not only parsing fake data.

Also checked and we don't have a test that this JSON might be helpful instead of the current Model Factories that we use.

I think the JSON generator might only have extra value if we select some real cases so we can generate multiple JSON objects for webhook testing but based on real data.